docker-compose.yml

```diff
@@ -27,6 +27,8 @@ services:
 
   django:
     build: .
+    environment:
+    - PYTHONUNBUFFERED=0
```

Comment:
I'm going to move this to the Dockerfile since it's not a setting that we're going to want to change per environment.